src/alchemtest/amber/access.py

```diff
@@ -96,12 +94,58 @@ def load_invalidfiles():
         - 'data' : the example of invalid data files
         - 'DESCR': the full description of the dataset
 
+
+    .. deprecated:: 0.7
+       use :func:`load_testfiles` instead
+
     """
 
-    module_path = dirname(__file__)
-    data = [glob(join(module_path, 'invalidfiles/*.out.bz2'))]
+    import warnings  # NOTE: this import is here and not where it should be just because we'll remove this function in next version
```

Comment:
Put the `import warnings` at the top of the file with all other imports, we will remove it from there when we remove the deprecated functionality.